common/app/views/main.scala.html

```diff
@@ -67,37 +67,6 @@
                     @fragments.immersiveGalleryMainMedia(page)
                 </div>
             }
-            case page: model.ContentPage if (page.item.content.isExplore && !page.item.content.tags.isInteractive && !page.item.content.tags.isVideo) => {
```

Comment:
Can someone confirm that an article cannot be an interactive or a video (both served by `applications`)?

Reply:
I believe it is true that the content type video and interactive cannot be the content type article. 

@blongden73 will be able to answer specific questions about explore though